tests/lib/exporters/timesketch_tam.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests the Threaded Timesketch exporter."""
+
+import unittest
+
+import mock
+
+from dftimewolf import config
+from dftimewolf.lib import state
+from dftimewolf.lib.containers import containers
+from dftimewolf.lib.exporters import timesketch_tam as timesketch
+from dftimewolf.lib import errors
+
+
+class TimesketchExporterThreadedTest(unittest.TestCase):
+  """Tests for the Threaded Timesketch exporter."""
+
+  def testInitialization(self):
+    """Tests that the processor can be initialized."""
+    test_state = state.DFTimewolfState(config.Config)
+    timesketch_exporter = timesketch.TimesketchExporterThreaded(test_state)
+    self.assertIsNotNone(timesketch_exporter)
+
+  # pylint: disable=invalid-name
+  @mock.patch('dftimewolf.lib.timesketch_utils.GetApiClient')
+  def testSetupWithSketchId(self, mock_GetApiClient):
+    """Tests the SetUp function."""
+    mock_sketch = mock.Mock()
+    mock_sketch.id = 1234
+    mock_sketch.my_acl = ['write']
+    mock_api_client = mock.Mock()
+    mock_api_client.get_sketch.return_value = mock_sketch
+    mock_GetApiClient.return_value = mock_api_client
+    test_state = state.DFTimewolfState(config.Config)
+    timesketch_exporter = timesketch.TimesketchExporterThreaded(test_state)
+    timesketch_exporter.SetUp(
+        incident_id=None,
+        sketch_id=1234,
+        analyzers=None
+    )
+    self.assertEqual(timesketch_exporter.sketch_id, 1234)
```

Comment:
Mega-nit: self.assertEqual's first argument is the expected value AFAIK, so this should be inverted to `self.assertEqual(1234, ...)`

Reply:
Hmm the unittest docs [1] only list the args as `first` and `second` as opposed to expected and actual.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual